Plugins/Public/base_plugin/PlayerCommands.cpp

```diff
@@ -1908,43 +1908,48 @@ namespace PlayerCommands
 	bool CheckSolarDistances(uint client, uint systemID, Vector pos)
 	{
 		// Other POB Check
-		for (const auto& base : player_bases)
-		{
-			if (base.second->basetype != "legacy" 
-				|| base.second->invulnerable 
-				|| !base.second->logic
-				|| base.second->system != systemID
-				|| (base.second->position.x == pos.x
-					&& base.second->position.y == pos.y
-					&& base.second->position.z == pos.z))
-				continue;
-
-			float distance = HkDistance3D(pos, base.second->position);
-			if (distance < minOtherPOBDistance)
+		if (minOtherPOBDistance > 0)
+		{
+			for (const auto& base : player_bases)
 			{
-				if (client)
-				{
-					PrintUserCmdText(client, L"Player base %ls is too close! Minimum distance: %um", base.second->basename.c_str(), static_cast<uint>(minOtherPOBDistance));
-				}
-				else
+				if (base.second->basetype != "legacy"
+					|| base.second->invulnerable
+					|| !base.second->logic
+					|| base.second->system != systemID
+					|| (base.second->position.x == pos.x
+						&& base.second->position.y == pos.y
+						&& base.second->position.z == pos.z))
+					continue;
+
+				float distance = HkDistance3D(pos, base.second->position);
+				if (distance < minOtherPOBDistance)
 				{
-					ConPrint(L"Base is too close to another Player Base, distance %um, name %ls", static_cast<uint>(distance), base.second->basename.c_str());
+					if (client)
+					{
+						PrintUserCmdText(client, L"Player base %ls is too close! Minimum distance: %um", base.second->basename.c_str(), static_cast<uint>(minOtherPOBDistance));

```

Comment:
Probably worth printing their current distance. There can be a misalignment of what the game's client/camera sees and what the server sees. Also you should format these numbers with commas and whatnot so they are readable. Reading that you are 3434231 to close would be very confusing.

Reply:
5 digits are plenty readable imo. Adding current distances tho.